pandas/core/strings.py

```diff
@@ -2370,6 +2370,16 @@ def cat(self, others=None, sep=None, na_rep=None, join=None):
     0          this is a regular sentence        None
     1  https://docs.python.org/3/tutorial  index.html
     2                                 NaN         NaN
+
+    Remember to escape special characters when explicitly using regular
+    expressions.
+
+    >>> s = pd.Series(["1+1=2", np.nan])
+
+    >>> s.str.split("\\+|=", expand=True)
```

Comment:
Could you use a raw string here instead? That would be more idiomatic to Python

Reply:
```suggestion
    >>> s.str.split(r"\+|=", expand=True)
```
I tried doing it this way but PyCharm's PEP8 checker still detects the `\+` as an invalid escape sequence. Should I just try force-pushing and checking if it will pass the `pep8speaks` check?

Reply:
Update: I've force-pushed an edit that applies the changes I've outlined above. 
@WillAyd `pep8speaks` threw out a W605 complaining about the invalid escape sequence. Do I keep these changes or do I revert it back to the `"\\+|="` string?

Reply:
Hmm OK that's strange. Can you check the code base for how we've handled elsewhere? I'd be surprised if this is the first time we've seen this

Reply:
I turned the entire docstring into a raw string by adding an `r` before the triple quotation marks at the beginning. This removed the PEP8 error both in my local machine and from `pep8speaks`. Kindly review. Thanks!